cmake/FindLibBpf.cmake

```diff
@@ -82,6 +82,10 @@ int main(void) {
 CHECK_CXX_SOURCE_COMPILES("
 #include <bpf/bpf.h>
 
+#if !defined(BPF_F_KPROBE_MULTI_RETURN)
+# error \"Undefined BPF_F_KPROBE_MULTI_RETURN\"
+#endif
```

Comment:
It'd be much nicer if you used `BPF_F_KPROBE_MULTI_RETURN` in the program itself. E.g.:
```
opts.kprobe_multi.flags = BPF_F_KPROBE_MULTI_RETURN;
```